code/modules/cargo/packs.dm

```diff
@@ -2284,6 +2349,48 @@
 					)
 	crate_name = "religious supplies crate"
 
+/datum/supply_pack/misc/lewd
+	name = "Lewd Crate" // OwO
+	desc = "Psss want to have a good time with your sluts? Well I got what you want maid clothing, dildos, collars and more!"
+	cost = 5000
+	contraband = TRUE
+	contains = list(/obj/item/dildo/custom,
+					/obj/item/dildo/custom,
+					/obj/item/vending_refill/kink,
+					/obj/item/vending_refill/kink,
+					/obj/item/clothing/under/maid,
+					/obj/item/clothing/under/maid,
+					/obj/item/electropack/shockcollar,
+					/obj/item/electropack/shockcollar,
+					/obj/item/restraints/handcuffs/fake/kinky,
+					/obj/item/restraints/handcuffs/fake/kinky,
+					/obj/item/clothing/head/kitty/genuine, // Why its illegal
+					/obj/item/clothing/head/kitty/genuine,
+					/obj/structure/reagent_dispensers/keg/aphro)
+	crate_name = "lewd kit"
+	crate_type = /obj/structure/closet/crate
+
+/datum/supply_pack/misc/lewdkeg
+	name = "Lewd Deluxe Keg" 
+	desc = "That other stuff not getting you ready? Well I have a Chemslut making tons of the good stuff."
+	cost = 7000 //It can be a weapon
+	contraband = TRUE
+	contains = list(/obj/structure/reagent_dispensers/keg/aphro/strong)
+	crate_name = "deluxe keg"
+	crate_type = /obj/structure/closet/crate
+
+/datum/supply_pack/misc/head_keys
```

Comment:
Ditto for the above comment

Reply:
people cryo with them all the time and then "Woops" someone one dies and there key's gone and or headset. This is for that or for borgs